src/app/InteractionModelJsonTlvHelper.h

```diff
@@ -0,0 +1,63 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
```

Comment:
Copyright (c) 2023 Project CHIP Authors

Reply:
this file has been removed, not planned for this pr, thanks